elasticsearch-twitter-webapp/app/search/search.js

```diff
@@ -4,14 +4,28 @@ angular.module('demo.search', ['ngRoute', 'elasticui'])
 
     .config(['$routeProvider', function($routeProvider) {
         $routeProvider.when('/search', {
-            templateUrl: 'search/search.html',
-            controller: 'SearchCtrl'
+            templateUrl: 'search/search.html'
         });
     }])
 
-    .controller('SearchCtrl', [function() {
+    .controller('SearchCtrl', ['$scope', '$timeout', function($scope, $timeout) {
 
+        // Function to get the data
+        $scope.getData = function(){
+          $scope.indexVM.refresh();
+        };
+
+        // Function to replicate setInterval using $timeout service.
+        $scope.intervalFunction = function(){
+          $timeout(function() {
+            $scope.getData();
+            $scope.intervalFunction();
+          }, 9000);
```

Comment:
It was 9 before. A multiple of 5 does seem more reasonable, though. Any preference?